src/atoms/Identicon/Identicon.tsx

```diff
@@ -1,6 +1,5 @@
 import makeBlockie from 'ethereum-blockies-base64';
 import React from 'react';
-
```

Comment:
We should preserve these extra lines, they're used to separate external dependencies like react from internal dependencies like files in src.